src/useExpandable/useExpandable.stories.tsx

```diff
@@ -33,8 +36,8 @@ const ExpandableList = ({ expandedKeys }: any) => {
 }
 
 interface ItemProps {
-  data: [TreeListItem<FakeRecord>[], Helpers<FakeRecord>],
-  index: number,
+  data: [TreeListItem<FakeRecord>[], Helpers<FakeRecord>]
+  index: number
   style: object
 }
```

Comment:
This is going to break the stories. Storybook requires commas or semicolons in interfaces in stories.

Reply:
Oh shit. Would it be easy to add a rule to our eslint config to enforce this? These changes were part of autofixes lint errors.